src/main/java/com/parsing/parsers/pdf/parsing/ManyTableDetector.java

```diff
@@ -0,0 +1,151 @@
+package com.parsing.parsers.pdf.parsing;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+@Component
+@Slf4j
+public class ManyTableDetector {
```

Comment:
TableDetector?

Reply:
We already had TableDetector.

Reply:
Let's discuss it tomorrow.

Reply:
Decided to make this TableDetector and TableRecognizer

Reply:
Done.